photon-core/src/main/java/org/photonvision/vision/target/TrackedTarget.java

```diff
@@ -126,11 +124,9 @@ public TrackedTarget(
         tvec.put(
                 0,
                 0,
-                new double[] {
-                    bestPose.getTranslation().getX(),
-                    bestPose.getTranslation().getY(),
-                    bestPose.getTranslation().getZ()
-                });
+                bestPose.getTranslation().getX(),
+                bestPose.getTranslation().getY(),
+                bestPose.getTranslation().getZ());
```

Comment:
Are you sure this is legal? Like what overload does this call?

Reply:
Is this not the correct method? `public int put(int row, int col, double... data)`

Reply:
I guess if unit tests pass send it. I don't see any javadocs online telling me its wrong at least